docs/src/content/docs/commands/GETRANGE.MD

```diff
@@ -0,0 +1,107 @@
+---
+title: GETRANGE
+description: The `GETRANGE` command in DiceDB is used get a substring of string provided the start and end.
+---
+The `GETRANGE` command in DiceDB is used get a substring of string provided the start and end.
+
+## Syntax
+
+```bash
+GETRANGE key start end
+```
+
+## Parameters
+| Parameter | Description                                                               | Type    | Required |
+|-----------|---------------------------------------------------------------------------|---------|----------|
+| `key`     | The name of the key to be set.                                            | String  | Yes      |
+| `start`   | The starting index of the required substring                              | Integer | Yes      |
+| `end`     | The ending index of the required substring.                               | Integer | Yes      |
+## Return values
+
+| Condition                                      | Return Value                                      |
+|------------------------------------------------|---------------------------------------------------|
+| if key contains a valid stirng                 | a substring based on the start and end indices    |
+| if `start` is greater than `end`               | an empty string is returned                       |
+| if the `end` exceeds the length of the string present at `key` | the entire string is returned     |
+| if the `start` is greater than the length of the string | an empty string is returned.             |
+
+## Behaviour
+- If the specified key does not exist, the `GETRANGE` command returns an empty string. 
+- If `start` is greater than `end`, the `GETRANGE` command returns an empty string.
+- If `start` is not within the length of the string, the `GETRANGE` command returns an empty string.
+- `start` and `end` can be negative which removed `end + 1` characters from the other side of the string.
```

Comment:
Shall we change it to?
`Both start and end can be negative, which removes characters from the string, starting from the end + 1 position on the right.`